src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -513,10 +513,32 @@ public Void caseTemplatizedType(TemplatizedType type) {
             emit("[]");
             return null;
           }
+          Iterator<JSType> it = type.getTemplateTypes().iterator();
+          if ("Object".equals(referencedType.getReferenceName())) {
+            emit("{ [");
```

Comment:
If the type is not templatized, could we just emit `Object`? Always emitting `{ [key: string]: number }` is a bit misleading - there are usages that are intended to be used as pseudo-maps (`Object<string, SomeThing>`) and usages where `Object` is just the super type of everything (well, most things). I think the latter should be translated as just `Object`, so that TypeScript doesn't offer arbitrary indexing into it.